src/components/Calculator.js

```diff
@@ -0,0 +1,33 @@
+import '../App.css';
+import React from 'react';
+// eslint-disable-next-line
```

Comment:
@tresorsawasawa Hi.
I have a problem with that linter, that linter asks me for a function instead of a class that's why I  disabled that linter (well I understood that)
![Captura desde 2023-01-09 20-10-32](https://user-images.githubusercontent.com/113067895/211446339-2ca5d7ef-008e-4163-8249-e8f013507aaa.png)


May you help me with a suggestion to fix this problem, please?

Reply:
Hi @sambeck87

Sorry to answer later, I was off. And right now I am not working, I am so far away from working place. But once I get home I can see what to do